File: squares-of-a-sorted-array/squares-of-a-sorted-array.py
class Solution(object):
    def sortedSquares(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        ans = []
        
        for num in nums:
            ans.append(num**2)
        
        ans.sort()
        
        return ans

File: squares-of-a-sorted-array/squares-of-a-sorted-array.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        int l = nums.length;
        int[] numSq = new int[l];
        for(int i = 0; i < l; i++) {
            numSq[i] = (int)Math.pow(nums[i], 2);
        }
        Arrays.sort(numSq);
        return numSq;
    }
}